packages/api-fetch/src/index.ts

```diff
@@ -197,3 +199,5 @@ apiFetch.mediaUploadMiddleware = mediaUploadMiddleware;
 apiFetch.createThemePreviewMiddleware = createThemePreviewMiddleware;
 
 export default apiFetch;
+export { apiFetch };
```

Comment:
This named export is to fix #59087 as suggested [here](https://github.com/WordPress/gutenberg/issues/59087#issuecomment-2493599892).

Reply:
I'm wondering how it is going to look like in WordPress core after the bundling step. Currently, the default `apiFetch` gets assigned to `wp.apiFetch` global so you can call it as `wp.apiFetch( ... )`. The question is what happens with the regular export after webpack does its magic

Reply:
Named imports should end up as `wp.apiFetch.apiFetch`.

Reply:
It looks like it works correctly in Gutenberg because it's configured accordingly:

<img width="1412" alt="Screenshot 2024-12-03 at 09 03 46" src="https://github.com/user-attachments/assets/fb8b6787-ad9c-4648-aae0-62b50cdc7ba3">

<img width="689" alt="Screenshot 2024-12-03 at 09 05 35" src="https://github.com/user-attachments/assets/7b21b1c2-c38b-410e-96b2-0951deeb3027">

I would rather avoid including this change in this PR.

@sirreal might have also some insights on how to make it a dual package.

Reply:
> I would rather avoid including this change in this PR.

Sure, let me revert that bit and create another PR to deal with that issue after this gets merged.

Reply:
Done